pkg/gomod/release_meta_test.go

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
add build e2e flag?
put with other integration tests instead?
make a mock instead?

Reply:
the only reason I did not was our e2e tests are limited to the `./test` dir and I want these tests to be local to the file they are testing, and mocking is an extreme amount of work